DOCS/license-inclusion-principles.md

```diff
@@ -4,30 +4,37 @@ Determining whether a candidate license should be included on the SPDX License L
 ## Candidate License Analysis
 
 **Definitive factors:**
-* The submitted license must not match another license already on the SPDX License List as per the SPDX matching guidelines.
-* All OSI-approved licenses will be included on the SPDX License List.
-  * This is consistent with early collaboration between SPDX and OSI (2011) to ensure that all OSI-approved licenses were represented on the SPDX License List. Also at that time, OSI began using the SPDX license identifiers in their URLs and on the license pages.
-* Software licenses that apply only to executables and do not provide for the availability of the source code will not be included on the SPDX License List.
-* The license has identifiable and stable text; it is not in the midst of drafting.
-* The license steward, if any, is committed to not modifying after addition to the list and to versioning new versions in the future.
+<ol type="A">
+ <li>The submitted license must not match another license already on the SPDX License List as per the SPDX matching guidelines.</li>
+ <li>All OSI-approved licenses will be included on the SPDX License List.</li>
+ <li>This is consistent with early collaboration between SPDX and OSI (2011) to ensure that all OSI-approved licenses were represented on the SPDX License List. Also at that time, OSI began using the SPDX license identifiers in their URLs and on the license pages.</li>
```

Comment:
This should be indented, since it's a comment about the prior item -- not a top-level item itself. Or maybe the note should be moved down to the "Historical Background" section below because it's really not a factor, rather an explanation.